### PR TITLE
Add my new source generators

### PR DESCRIPTION
**Pull request by musictopia2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +101,7 @@ Template for entries:
...
 - [StructPacker](https://github.com/RudolfKurka/StructPacker)  - ![stars](https://img.shields.io/github/stars/RudolfKurka/StructPacker?style=flat-square&cacheSeconds=604800) ![last commit](https://img.shields.io/github/last-commit/RudolfKurka/StructPacker?style=flat-square&cacheSeconds=86400) binary serializer that auto-generates C# serialization code to achieve peak runtime performance and efficiency.
 - [Svg to C# Source Generators](https://github.com/wieslawsoltes/Svg.Skia)  - ![stars](https://img.shields.io/github/stars/wieslawsoltes/Svg.Skia?style=flat-square&cacheSeconds=604800) ![last commit](https://img.shields.io/github/last-commit/wieslawsoltes/Svg.Skia?style=flat-square&cacheSeconds=86400) SVGC compiles SVG drawing markup to C# using SkiaSharp as rendering engine. SVGC can be also used as codegen for upcoming C# 9 Source Generator feature.
+- [TxtToListGenerator](https://github.com/musictopia2/TxtToListGenerator)  - A source generator where if you have a text file as additional file and you have a list ordered by return carriages, then it produces either a list of int or a list of string in c#.
```
Comment (amis92) [inline README.md:104]: This is not the correct order, should be after Tinyhand.

**Review by amis92 — CHANGES_REQUESTED** [review]
Please follow the template preceding the list, and add the stars/last-commit badges.

**Comment by musictopia2** [comment]
How do I add the stars and last commit badges?  I am guessing if I copied from the previous item, it would show the previous item's stars and last commit which is not accurate.

**Comment by amis92** [comment]
There's a template at https://github.com/amis92/csharp-source-generators/blob/06b7d5e59b4f2c00f4c0bec6c5b7d4cf6b57d91b/README.md?plain=1#L20-L23 in which you'd replace `REPO` with `user/repo` part of GitHub URL and append the description. :)

**Comment by musictopia2** [comment]
I went ahead and submitted a new pull request.  Because I did not know how to resolve the conflicts without making a backup copy then reforked it.

**Pull request closed without merging**